docs/ANONYMITY_NETWORKS.md

```diff
@@ -1,149 +1,175 @@
-(via sybil) more difficult.
 
+Currently Tor/I2P is available. Anonymity networks (AN) wrap traffic and proxy it over several hops to the final destination, which can include some exit back into clearnet or be a hidden service inside the AN itself. The support is experimental and while anonymity networks conceal node information there will be corner-cases that could leak information.
 
-## Behavior
 
-If _any_ anonymity network is enabled, transactions being broadcast that lack
-a valid "context" (i.e. the transaction did not come from a p2p connection),
-will only be sent to peers on anonymity networks. If an anonymity network is
-enabled but no peers over an anonymity network are available, an error is
-logged and the transaction is kept for future broadcasting over an anonymity
-network. The transaction will not be broadcast unless an anonymity connection
-is made or until `monerod` is shutdown and restarted with only public
-connections enabled.
+## Table of Contents
 
-Anonymity networks can also be used with `monero-wallet-cli` and
-`monero-wallet-rpc` - the wallets will connect to a daemon through a proxy. The
-daemon must provide a hidden service for the RPC itself, which is separate from
-the hidden service for P2P connections.
+  - [Outbound connections](##Outbound-connections)
+     - [Proxy node traffic](###Proxy-node-traffic)
+    - [Connect to hidden services](###Connect-to-hidden-service)
+    - [Peer handling](###Peer-handling)
+  - [Inbound connections](##Inbound-connections)
+    - [Allow inbound RPC](###Allow-inbound-RPC-(Server))
+    - [Connect to remote daemon](###Connect-to-remote-daemon-(Client))
+  - [Default configurations examples](##Default-configurations)
+  - [Privacy Limitations](##Privacy-Limitations)
+  - [Using Tor on tails (whonix)](##Using-Tor-on-tails-(whonix))
+  - [Legacy options](##Legacy-options)
+    - [Wrapping-monerod-with-torsocks](###-Wrapping-monerod-with-torsocks-(now:-use---proxy))
+  
+Three major options configure `monerod` and differ in connection, traffic type and route:
 
+| Connection | Type   | Clearnet | Exit-Relay/I2PTunnel | .onions/.i2p        |
+|------------|--------|----------|----------------------|---------------------|
+| incoming   | tx     | default  |           -          | --anonymous-inbound |
+|            | blocks | default  |           -          | N/A                 |
+| outgoing   | tx     | default  | --proxy              | --tx-proxy          |
+|            | blocks | default  | "                    | N/A                 | 
 
-## P2P Commands
+Clearnet (Ip4) is the default. Connections are hopened both ways and peers sync blocks and tx (handshakes, peer timed syncs and transactions). Setting up an AN with `--proxy`, relays blocks and peer transactions (p2p) over relays. Introducing hidden services for transactions increases anonymity. You can provide a service yourself `--anonymous-inbound` or connect to the hidden service of a peer.
```

Comment:
My understanding was that Dandelion++ nodes only send transactions on outgoing connections. If --proxy is set (and clearnet is bound to localhost, which I think is what most people would intend) then the node will have no incoming transactions and so only non-Dandelion++ nodes would ever relay transactions to it. Stating that --proxy impacts transaction relaying is a bit misleading if this is the case.